internal/services/v1/schema.go

```diff
@@ -35,15 +35,17 @@ func NewSchemaServer(additiveOnly bool) v1.SchemaServiceServer {
 				usagemetrics.StreamServerInterceptor(),
 			),
 		},
-		additiveOnly: additiveOnly,
+		additiveOnly:        additiveOnly,
+		expiringRelsEnabled: expiringRelsEnabled,
```

Comment:
super nit: maybe invert this flag, since enabled is the default?

Reply:
I prefer to leave it as "is this enabled"